src/ai-bundle/config/options.php

```diff
@@ -160,6 +160,7 @@
                                 ->defaultValue('http_client')
                                 ->info('Service ID of the HTTP client to use')
                             ->end()
+                            ->booleanNode('use_api_as_catalog')->end()
```

Comment:
```suggestion
                            ->booleanNode('api_catalog')->end()
```
?
and add a description what it does and what can be side effects

Also update the docs please